src/main/java/org/opendatakit/suitcase/utils/FileUtils.java

```diff
@@ -123,15 +123,17 @@ public static Path getInstancesPath(CloudEndpointInfo cloudEndpointInfo, String
     return getBasePath(cloudEndpointInfo, tableId, savePath).resolve(INSTANCES_PATH);
   }
 
-  public static String getCSVName(CsvConfig config) {
+  public static String getCSVName(CsvConfig config, String tableId) {
     StringBuilder csvNameBuilder = new StringBuilder();
 
     csvNameBuilder
-        .append(config.isDownloadAttachment() ? DATA_CSV_MODIFIER : LINK_CSV_MODIFIER);
-
-    csvNameBuilder
-        .append(CSV_MODIFIER_SEPARATOR)
-        .append(config.isScanFormatting() ? FORMATTED_CSV_MODIFIER : UNFORMATTED_CSV_MODIFIER);
+        .append(config.isDownloadAttachment() ? DATA_CSV_MODIFIER : tableId);
```

Comment:
Now looking at how the code works, it is best to leave the LINK_CSV_MODIFIER name and instead simply append the table tableId before line 128 and append a CSV_MODIFIER_SEPARATOR.

So after line 127 it should be something like "csvNameBuilder.append(tableId).append(CSV_MODIFIER_SEPARATOR).

Reply:
Sure !! will make the changes